content/modules/ROOT/pages/m3/module-03.6.adoc

```diff
@@ -11,6 +11,8 @@ While the developer can add the necessary annotation to their Component, it's be
 
 === Login as Platform Engineer
 
+IMPORTANT: Please ensure you are logged in as a *Platform Engineer* 
```

Comment:
```suggestion
```

Reply:
I think this is unnecessary, since the section explicitly guides them through it.